.github/workflows/main.yml

```diff
@@ -41,50 +70,104 @@ jobs:
         run: |
           wget -q https://github.com/pyodide/pyodide/releases/download/${{ matrix.pyodide-version }}/pyodide-build-${{ matrix.pyodide-version }}.tar.bz2
           tar xjf pyodide-build-${{ matrix.pyodide-version }}.tar.bz2
-          mv pyodide dist/
+          mv pyodide pyodide-dist/
+  make_test_matrix:
+    outputs:
+      matrix: ${{ steps.filter.outputs.matrix }}
+      pyodide_versions: ${{ steps.filter.outputs.pyodide_versions }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Write py file
+        run: |
+          cat << 'EOF' > filtermatrix.py
```

Comment:
Can we actually have this as a standalone .py file, or does it need to be part of the .yaml to work when called from other actions?